tests/unit/amo/helpers.js

```diff
@@ -347,7 +347,7 @@ export function dispatchAutocompleteResults({
 
 export const createFakeCollectionDetail = ({
   name = 'My Addons',
-  count = 123,
+  count = 30,
```

Comment:
That will likely break other tests. I think we should override `count` in `createFakeCollectionDetail()`, just for this test case.

Reply:
oh ok sure..